src/entities/supportRequest.ts

```diff
@@ -8,15 +8,25 @@ export enum SupportRequestErrors {
   ExistingActiveRequest = 'ExistingActiveRequest',
 }
 
+export interface SupportRequestExtraData {
+  problemDescription: string;
+}
+
 @Entity()
 export class SupportRequest extends BaseEntity {
-  constructor(slackIdId: string, name: string, type: SupportRequestType, primaryLanguage = '', problemDescription = '') {
+  constructor(
+    slackIdId: string,
+    name: string,
+    type: SupportRequestType,
+    primaryLanguage = '',
+    extraData: SupportRequestExtraData = { problemDescription: '' },
+  ) {
     super();
 
     this.slackId = slackIdId;
     this.name = name;
     this.primaryLanguage = primaryLanguage;
-    this.extraData = { problemDescription };
+    this.extraData = extraData;
```

Comment:
Since `extraData` is now nullable we can default it to null which will make adding new items easier too

```suggestion
  constructor(
    slackIdId: string,
    name: string,
    type: SupportRequestType,
    primaryLanguage = '',
    extraData: SupportRequestExtraData = null,
  ) {
    super();
    this.slackId = slackIdId;
    this.name = name;
    this.primaryLanguage = primaryLanguage;
    this.extraData = { problemDescription };
    this.extraData = extraData;
```